azblob/list.go

```diff
@@ -68,7 +84,7 @@ func (azp *Storer) FilteredList(ctx context.Context, tagsFilter string, opts ...
 	}
 
 	if options.listMarker != nil {
-		span.SetTag("marker", *options.listMarker)
+		span.SetTag("marker", options.listMarker)
```

Comment:
Why did you change this to set the ptr value ? We know it is not nil, we want to know its value don't we ?

Reply:
mistake - fixed